server/src/middleware/csrf.js

```diff
@@ -26,7 +26,8 @@ function isCsrfExemptPath(path) {
   return isAuthPath(path)
     || path.startsWith("/data")
     || path === "/event"
-    || path === "/error";
+    || path === "/error"
+    || path === "/api/set-login-cookie";
```

Comment:
If the POST `/api/set-login-cookie` sends an origin header, we should validate its protocol/scheme is `moz-extension://` in the csrf middleware.

Reply:
It appears not to send any Origin header at all. I'm a bit surprised, I would have expected it to be the site origin. Or maybe it would only be set for CORS requests?